src/grandorgue/GOSoundEngine.cpp

```diff
@@ -576,27 +596,39 @@ void GOSoundEngine::CreateReleaseSampler(GO_SAMPLER* handle)
 					}
 				}
 			}
-			unsigned cross_fade_len = this_pipe->GetReleaseCrossfadeLength();
-			new_sampler->fader.NewAttacking(gain_target, cross_fade_len, m_SampleRate);
 
-			if (m_ReleaseLength > 0)
-			{
-				if ( m_ReleaseLength < gain_decay_length || gain_decay_length == 0 )
-					gain_decay_length = m_ReleaseLength;
-			}
-
-			if (gain_decay_length > 0)
-				new_sampler->fader.StartDecay(gain_decay_length, m_SampleRate);
 
-			if (m_ReleaseAlignmentEnabled && release_section->SupportsStreamAlignment())
-			{
-				release_section->InitAlignedStream(&new_sampler->stream, &handle->stream);
-			}
-			else
-			{
-				release_section->InitStream(&m_ResamplerCoefs, &new_sampler->stream, this_pipe->GetTuning() / (float)m_SampleRate);
-			}
-			new_sampler->is_release = true;
+            /*  Algorithm to Choose Which Release Truncation or Scaling Method to activate. */
```

Comment:
It is more clear to move this comment and assignment to truncation_fade_len below calling NewAttacking